src/alembic/versions/edadb97f6502_initial_state.py

```diff
@@ -0,0 +1,385 @@
+    sa.Column('exon_set_id', sa.Integer(), nullable=False),
+    sa.Column('start_i', sa.Integer(), nullable=False),
+    sa.Column('end_i', sa.Integer(), nullable=False),
+    sa.Column('ord', sa.Integer(), nullable=False),
+    sa.Column('name', sa.Text(), nullable=True),
+    sa.CheckConstraint('start_i < end_i', name='exon_start_i_must_be_lt_end_i'),
+    sa.ForeignKeyConstraint(['exon_set_id'], ['uta.exon_set.exon_set_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('exon_id'),
+    sa.UniqueConstraint('exon_set_id', 'end_i', name='end_i_must_be_unique_in_exon_set'),
+    sa.UniqueConstraint('exon_set_id', 'start_i', name='start_i_must_be_unique_in_exon_set'),
+    schema='uta'
+    )
+    op.create_index(op.f('ix_uta_exon_exon_set_id'), 'exon', ['exon_set_id'], unique=False, schema='uta')
+    op.create_table('exon_aln',
+    sa.Column('exon_aln_id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('tx_exon_id', sa.Integer(), nullable=False),
+    sa.Column('alt_exon_id', sa.Integer(), nullable=False),
+    sa.Column('cigar', sa.Text(), nullable=False),
+    sa.Column('added', sa.DateTime(), nullable=False),
+    sa.Column('tx_aseq', sa.Text(), nullable=True),
+    sa.Column('alt_aseq', sa.Text(), nullable=True),
+    sa.ForeignKeyConstraint(['alt_exon_id'], ['uta.exon.exon_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['tx_exon_id'], ['uta.exon.exon_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('exon_aln_id'),
+    schema='uta'
+    )
+    op.create_index(op.f('ix_uta_exon_aln_alt_exon_id'), 'exon_aln', ['alt_exon_id'], unique=False, schema='uta')
+    op.create_index(op.f('ix_uta_exon_aln_tx_exon_id'), 'exon_aln', ['tx_exon_id'], unique=False, schema='uta')
+    # ### end Alembic commands ###
+
+    # ### custom commands to match the initial UTA database schema 1.1 ###
+    op.create_table('associated_accessions',
+    sa.Column('associated_accession_id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('tx_ac', sa.Text(), nullable=True),
+    sa.Column('pro_ac', sa.Text(), nullable=True),
+    sa.Column('origin', sa.Text(), nullable=False),
+    sa.Column('added', sa.TIMESTAMP(), server_default=sa.text('now()'), nullable=False),
+    sa.PrimaryKeyConstraint('associated_accession_id'),
+    schema='uta'
+    )
+    op.create_index('associated_accessions_tx_ac', 'associated_accessions', ['tx_ac'], unique=False, schema='uta')
+    op.create_index('associated_accessions_pro_ac', 'associated_accessions', ['pro_ac'], unique=False, schema='uta')
+    op.create_index('unique_pair_in_origin', 'associated_accessions', ['origin', 'tx_ac', 'pro_ac'], unique=True, schema='uta')
+    op.create_table_comment('associated_accessions', 'transcript-protein accession pairs associated in source databases', schema='uta')
+    # ### end custom commands ###
+
+    # ### custom SQL to add views to match the initial UTA database schema 1.1 ###
```

Comment:
Added this section to account for all existing views, materialized views, and comments.